### ISSUE TITLE
Release crx binary

### ISSUE THREAD

**Issue by aviaryan** [aashutoshrathi/Insta-Downloader-Extension#9]
Would it be possible to release a `crx` binary with this project?
This will allow anyone to quickly install the Chrome extension without going through the manual step of downloading, extracting and installing the zip.

**Comment by lalitmee** [comment]
I have created `.crx` binary for this project as well as `.pem` file also. Should I add? If you want I can upload here for confirmation.

**Comment by aashutoshrathi** [comment]
Yes, @lalitmee proceed please

**Comment by aashutoshrathi** [comment]
Done, but now after drag and drop install of crx, it says extension might have been corrupted.

**Comment by lalitmee** [comment]
Ok. Lemme check that where and what's the problem.

**Comment by lalitmee** [comment]
It's working. No dialog is appearing that extension might have been corrupted.

**Comment by lalitmee** [comment]
Ohk. You removed `pem` file. Yeah, I was thinking that if we can generate it every time then what's the need of having it there.

**Comment by aashutoshrathi** [comment]
Chrome actually prompted to remove .pem files.

**Comment by aashutoshrathi** [comment]
Sir, still facing same

![insta-error](https://user-images.githubusercontent.com/21199234/34470202-4c2572aa-ef52-11e7-93a6-60b1462163a5.gif)

**Comment by lalitmee** [comment]
But I am not getting this. See this.

![screenshot from 2018-01-02 00-18-38](https://user-images.githubusercontent.com/10762218/34470209-89a7abde-ef52-11e7-95c2-2115fe83e1c1.png)

**Comment by aashutoshrathi** [comment]
Do you think crx are OS specific?

**Comment by lalitmee** [comment]
Hey, check the last comment on this [page](https://productforums.google.com/forum/#!msg/chrome/49mvhGxGzxo/ityoMn1qBgAJ).

**Comment by lalitmee** [comment]
I don't think that `crx` are OS specific because all the work is done by Chrome browser, not the OS. Still, we can check on google. 

Are you using the latest version of Google Chrome?

**Comment by aashutoshrathi** [comment]
Yes, I just checked once more for updates.

**Comment by lalitmee** [comment]
Then why it is showing. It is not showing anything like that in mine. I have drag and dropped so many times. But no message like that. 
What do you say?

**Comment by aashutoshrathi** [comment]
No Idea
